Z2A SwiftUI/NSCoffee/NSCoffee/Views/DrinkDetail.swift

```diff
@@ -34,6 +37,44 @@ struct DrinkDetail: View {
             MilkTypeView()
         }
         .listStyle(.grouped)
+        .onChange(of: extras) { oldValue, newValue in
+            if newValue.count > 0 {
+                order = nil
```

Comment:
For some reason the order here will only update if I set it to nil first. Any ideas what I've done wrong?

Reply:
Oh, not sure. Should actually be nil by default already, shouldn't it?